apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -88,13 +112,46 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
         setActiveIndex(tempIndexArr);
     };
 
+    /**
+     * Handles the LogData View panel close
+    */
+    const handleLogDataViewClose= (): void => {
```

Comment:
End comments with a full stop. Please check other places as well.

```suggestion
    /**
     * Handles the LogData View panel close.
    */
    const handleLogDataViewClose = (): void => {
```